.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: elixir
+elixir:
+  - 1.4
```

Comment:
Why are we using `1.4`? `1.6` is out with some really cool features we could be using

Reply:
We could actually consider not using credo if we use `mix format` instead offered in (elixir 1.6)

Reply:
Is 1.6 available on Travis, I'm not sure :thinking: ?

Reply:
https://github.com/phoenixframework/phoenix/blob/master/.travis.yml#L5

Reply:
I think the reason why we don't have 1.6 is that the version on Heroku can be up to 1.5:
https://github.com/HashNuke/heroku-buildpack-elixir/blob/master/elixir_buildpack.config
So I would prefer to keep the same version between Heroku and Travis to make sure the tests and the app are run on the same version.
But at the end I think the version has been updated on the project since:
https://github.com/dwyl/github-backup/blob/7c7c2de83a6f2047925efa0da5ffd86dcaa855f2/.travis.yml#L3